scrapy_crawlera.py

```diff
@@ -54,7 +69,8 @@ def open_spider(self, spider):
             self.url += '?noconnect'
 
         self._proxyauth = self.get_proxyauth(spider)
-        logging.info("Using crawlera at %s (user: %s)" % (self.url, self.user))
+        logging.info("Using crawlera at %s (identity: %s)" % (
```

Comment:
I would also truncate the API key for security reasons. Showing just the first characters should be enough to identify the API key you're using.

For example, if your API key is:

    aab6a8700e3a9fa1cd6a3bc0a80a01c6

It could show the first 8 chars:

    aab6a870...

/cc @amleczko